README.md

```diff
@@ -2814,7 +2814,7 @@ To use SSL with a virtual host, you must either set the [`default_ssl_vhost`][]
 
 * `ssl_protocol`
 
-  Default: ['all', '-SSLv2', '-SSLv3', '-TLSv1', '-TLSv1.1'].
+  Default: ['all', '*SSLv2', '*SSLv3'].
```

Comment:
Probably because it's a straight revert this is changed back to `*` but can you fix this to be `-SSLv2` so it matches the default?